src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -7,10 +7,10 @@
     "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/bigquery.svg",
     "clients": [
       {
+      "name":"Google Cloud Java Client for BigQuery",
       "language":"java",
       "site":"https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-java",
-      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/0.22.0/apidocs/com/google/cloud/bigquery/package-summary.html",
-      "version" : "0.22.0",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/bigquery/package-summary.html",
```

Comment:
Would this be clearer as a language-specific field called `javadoc`?
Oh, actually we should have a `javadoc` field for programmatic access (e.g., Eclipse javadoc hovers) as well as a the more human-meaningful API starting point.

Reply:
what would the difference be. For example, what javadoc URL would Eclipse need here?

Reply:
When I try setting the Javadoc URL for a jar to this URL (right-click on a jar, like a bundle in the _Plug-in Dependencies_, and select _Javadoc Location_, and set as a URL), Eclipse shown an error:
<img width="633" alt="screen shot 2017-08-28 at 4 32 01 pm" src="https://user-images.githubusercontent.com/202851/29792449-72c70e2a-8c0e-11e7-8712-caea54f30fb3.PNG">

The java docs have a specific layout, as does the HTML.  So Eclipse is able to find and parse out the appropriate Javadoc to be shown in a hover.

Reply:
To make that work, I think we first need to get google-cloud-java to publish individual javadoc for each API. The file it's looking for is at

https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/

Reply:
We should probably pull IDE readable source and javadoc jars from Maven Central rather than calling them out as a separate  artifact.

Reply:
+1 to @elharo 's comment - then the javadoc and source will actually match the classes that the user is using.

Reply:
CT for IntelliJ plans to expose these links to the end user so it needs human browsable URLs.

The links to the Maven source/javadoc jars don't need to be explicitly included here because they can be derived from the Maven coordinates.

Reply:
Sounds good.